pocs/linux/kernelctf/CVE-2023-31436_mitigation/metadata.json

```
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://google.github.io/security-research/kernelctf/metadata.schema.v1.json",
+    "submission_id": "exp52",
```

In v2, this is an array:
```suggestion
    "submission_ids": ["exp52"],
```